### PR TITLE
geolocate endpoint, tests and sample app

### PR DESCRIPTION
**Pull request by pangea-andrest**

### CODE REVIEW

File: packages/pangea-sdk/pangea/services/intel.py [hunk 1]
```diff
@@ -421,32 +398,39 @@ def lookup(
...
         input = IPLookupRequest(ip=ip, verbose=verbose, raw=raw, provider=provider)
         response = self.request.post("lookup", data=input.dict(exclude_none=True))
         response.result = IPLookupResult(**response.raw_result)
         return response
 
+    def geolocate(
+        self, ip: str, verbose: Optional[bool] = None, raw: Optional[bool] = None, provider: Optional[str] = None
+    ) -> PangeaResponse[IPGeolocateResult]:
+        """
+        Retrieve information about the location of an IP address.
```
Comment (r3dcrosse) [inline packages/pangea-sdk/pangea/services/intel.py:411]:
```suggestion
        Geolocate an IP
    
        Retrieve information about the location of an IP address.
```

File: packages/pangea-sdk/pangea/services/intel.py [hunk 2]
```diff
@@ -421,32 +398,39 @@ def lookup(
...
+
+        Args:
+            ip (str): ip to request for a geolocate
```
Comment (r3dcrosse) [inline packages/pangea-sdk/pangea/services/intel.py:414]:
```suggestion
            ip (str): IP address to be geolocated
```

File: packages/pangea-sdk/pangea/services/intel.py [hunk 3]
```diff
@@ -421,32 +398,39 @@ def lookup(
...
+            provider (str, optional): intel provider to perfome lookup
```
Comment (r3dcrosse) [inline packages/pangea-sdk/pangea/services/intel.py:415]:
```suggestion
            provider (str, optional): Use geolocation data from this provider ("digitalenvoy"). Default provider defined by the configuration.
```

File: packages/pangea-sdk/pangea/services/intel.py [hunk 4]
```diff
@@ -421,32 +398,39 @@ def lookup(
...
+            verbose (bool, optional): true to get more detalied response
```
Comment (r3dcrosse) [inline packages/pangea-sdk/pangea/services/intel.py:416]:
```suggestion
            verbose (bool, optional): Echo the API parameters in the response
```

File: packages/pangea-sdk/pangea/services/intel.py [hunk 5]
```diff
@@ -421,32 +398,39 @@ def lookup(
...
+            raw (bool, optional): true to get provider raw response
```
Comment (r3dcrosse) [inline packages/pangea-sdk/pangea/services/intel.py:417]:
```suggestion
            raw (bool, optional): Include raw data from this provider
```

File: packages/pangea-sdk/pangea/services/intel.py [hunk 6]
```diff
@@ -410,7 +384,10 @@ def lookup(
         Retrieve IP address reputation from a provider.
```
Comment (r3dcrosse) [inline packages/pangea-sdk/pangea/services/intel.py:384]:
```suggestion
        Look up an IP

        Retrieve a reputation score for an IP address from a provider, including an optional detailed report.

        Args:
            input (IPLookupInput): input with IP information to perform request
            verbose (bool, optional): Echo the API parameters in the response
            raw (bool, optional): Include raw data from this provider
            provider (str, optional): Use reputation data from this provider: "crowdstrike"
```

Comment (pangea-andrest) [reply packages/pangea-sdk/pangea/services/intel.py]: Updated!

**Comment by pangea-andrest** [comment]
@r3dcrosse updated all suggestions. Thanks!

**Review by r3dcrosse — APPROVED** [review]

**Pull request merged**